### PR TITLE
Added initial content for the web console

### PR DESCRIPTION
**Pull request by diawarr**
- Added a parent topic to act as introductory content for the web console
- Added generic instructions for creating a service so that the steps can be reused/linked from product-specific instructions
- Added instructions for creating an authentication token (these seemed to be missing in the CLI and API topics, but this could be then linked in those)

I didn't add a separate how-to folder or navigation level for the tasks, as it seemed a bit unnecessary for these few topics, but those can be added later if there is a need for more content within this section.

### CODE REVIEW

**Review by hab-anka — APPROVED** [review]
Looks good, thanks. Happy to reuse some of these.

**Comment by lornajane** [comment]
I think we should wait until we can merge #73 and then decide if this content could be in the same structure. This seems like fairly reusable platform-level content to me.

**Comment by lornajane** [comment]
Now that #73 has been merged, @diawarr could you please rebase this branch onto main and move the files into the Platform section? (let me know if I can help with the rebase, or if you have questions)

**Comment by diawarr** [comment]
@lornajane - yep, that's now moved to the platform section ;)

File: docs/platform/howto/create_new_service.rst
Comment (lornajane) [inline docs/platform/howto/create_new_service.rst:24]: The note should be an admonition

File: docs/platform/howto/create_new_service.rst
Comment (lornajane) [inline docs/platform/howto/create_new_service.rst:46]: Is the upper-case, italic formatting indicating something in particular? I'm not sure I've seen this before and it looks a little out of place.

File: docs/platform/howto/create_new_service.rst
Comment (lornajane) [inline docs/platform/howto/create_new_service.rst:18]: These seem to render as numbers, rather than letters, is that a problem?

File: docs/platform/concepts/console.rst
Comment (lornajane) [inline docs/platform/concepts/console.rst:2]: Since we no longer have the separate console section, I don't think we need to have the landing page for the console either.

**Review by lornajane — CHANGES_REQUESTED** [review]
Thanks! I think this is close, I had a couple of small suggestions and also one page to remove as a result of the change in content structure since you originally created this branch.

File: docs/platform/howto/create_authentication_token.rst
Comment (diawarr) [reply docs/platform/howto/create_authentication_token.rst]: True, that's updated now ;)

File: docs/platform/howto/create_new_service.rst
Comment (diawarr) [reply docs/platform/howto/create_new_service.rst]: True, that's updated now ;)

File: docs/platform/howto/create_new_service.rst
Comment (diawarr) [reply docs/platform/howto/create_new_service.rst]: Yep, done ;)

File: docs/platform/howto/create_new_service.rst
Comment (diawarr) [reply docs/platform/howto/create_new_service.rst:48]: The capitalization is just based on how it appears in the UI - it might not look ideal in the text, but it's better to reference UI texts as they are in the UI, even though the capitalization there isn't very uniform...

File: docs/platform/howto/create_new_service.rst
Comment (diawarr) [reply docs/platform/howto/create_new_service.rst:18]: I guess that's something that could be solved in the CSS - it seems to be the case for all nested ol elements, but I don't think it's a huge issue ;)

File: docs/platform/concepts/console.rst
Comment (diawarr) [reply docs/platform/concepts/console.rst]: Yep, removed ;)

File: docs/platform/howto/create_new_service.rst
Comment (lornajane) [reply docs/platform/howto/create_new_service.rst:48]: I can't find anywhere it's capitalised when I use the console. Is there any danger this changed recently?
services-status2
services-status1

Comment (diawarr) [reply docs/platform/howto/create_new_service.rst:48]: Hmm... Maybe that's changed since I created the initial version of the content - that's now updated ;)

**Review by lornajane — APPROVED** [review]
We made it! Thanks :)

**Pull request merged**
